src/providers/web3.ts

```diff
@@ -1,4 +1,4 @@
 import Web3 = require('web3');
 
 const http = new Web3.providers.HttpProvider(`https://mainnet.infura.io/${process.env.INFURA_ID}`);
-export const web3 = new Web3(http);
+export const web3 = new Web3(http);
```

Comment:
Looks like the linting Git hook is not being called?

Reply:
Argh :|